src/preferences/dialog/dlgprefsound.cpp

```diff
@@ -303,6 +306,7 @@ DlgPrefSound::DlgPrefSound(QWidget* pParent,
 void DlgPrefSound::slotUpdate() {
     m_bSkipConfigClear = true;
     loadSettings();
+    settingChanged();
```

Comment:
@acolombier Why was this change necessary?
Thing is, with #12194, Apply will reapply the soundconfig unexpectedly for no obvious reason which causes a playback pause.

Reply:
This was necessary to update the dual threading components, but I guess it could have been in `loadSettings`